docker/Dockerfile

```diff
@@ -1,22 +1,25 @@
-FROM debian:jessie
+FROM frolvlad/alpine-glibc
```

Comment:
Makes sense.  Could a comment be added to the Dockerfile about this requirement or dependency?